### PR TITLE
Add tsantos serializer recipe

### PR DESCRIPTION
**Pull request by tsantos84**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: tsantos/serializer-bundle/1.0/config/packages/tsantos_serializer.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+tsantos_serializer:
+    generation_strategy: file_not_exists
+#    mapping:
```
Comment (Nyholm) [inline tsantos/serializer-bundle/1.0/config/packages/tsantos_serializer.yaml]: Maybe it make more sense to uncomment these?

**Comment by Nyholm** [comment]
Thank you for this PR. Can you make sure Travis is happy? It seams like you have a dependency on a twig service.

File: tsantos/serializer-bundle/1.0/config/packages/tsantos_serializer.yaml (hunk 1 above)
Comment (tsantos84) [reply tsantos/serializer-bundle/1.0/config/packages/tsantos_serializer.yaml]: The comments are just templates to indicate how to configure such option.

**Comment by tsantos84** [comment]
Hi @Nyholm, TravisCI seems to be happy now.

File: tsantos/serializer-bundle/1.0/config/packages/tsantos_serializer.yaml (hunk 1 above)
Comment (Nyholm) [reply tsantos/serializer-bundle/1.0/config/packages/tsantos_serializer.yaml]: I do not prefer config in comments like this. I suggest the following: 
- Add a link to your bundles documentation
- Remove these comments OR (if it makes sense) un-comment these lines.

**Comment by Nyholm** [comment]
Great. I just had one suggestion.

**Comment by tsantos84** [comment]
Hi @Nyholm, the comments as been removed.

**Pull request merged**